list.rb

```diff
@@ -0,0 +1,16 @@
+require './enumarable'
```

Comment:
- [ ] Great job so far @Team. There is just one change I will suggest you implement and you'll be on your way to the next Milestone :wink: 
- [ ] Kindly ensure that you make use of `require_relative` to import `MyEnumerable` since using `require_relative` for files in your directory instead of `require` is a ruby convention. :100: 
- [ ] _"`Require` should be used for external files, like gems, while `require_relative` should be used for referring to files within your directory."_